### ISSUE TITLE
Request for newer Rstudio build for win-64

### ISSUE THREAD

**Issue by andreifoldes** [conda-forge/rstudio-feedstock#11]
### Comment:

Hello,

As per an issue filed here: https://github.com/conda-forge/r-sjplot-feedstock/issues/21#issuecomment-1268806791 I would like to request a newer RStudio build for win-64. When trying a workaround whereby I connect my desktop RStudio to a conda environment that only contains the "r-sjplot" install I got a dll error on win-64 (also didn't work on centos 7.6 where Rstudio started but didn't load any of the GUI element other than the menus on top). I think it would be important to have an RStudio that works with the sjplot feedstock out of the box.

**Comment by izahn** [comment]
I don't currently have access to a Windows computer which makes development harder. I also don't use windows and so am less motivated to work on it. All help and contributions will be appreciated.

**Comment by andreifoldes** [comment]
Understood. I believe the problem outlined also applies to Centos 7.6.

**Comment by izahn** [comment]
The `rstudio-desktop` package works for me on Centos 7. If it does not work for you please open an issue with reproduction steps.

**Comment by andreifoldes** [comment]
Dear izahn,  
I realized I was using the wrong feedstock by using conda install rstudio instead of rstudio-desktop. Apologies, now it does work indeed!

**Comment by janstrauss1** [comment]
Dear @izahn,

I fully understand your opinion:

> I don't currently have access to a Windows computer which makes development harder. I also don't use windows and so am less motivated to work on it. All help and contributions will be appreciated.

Yet, in my new job I found myself back in a Windows environment and would like to use conda to manage `R `and `Rstudio`. Thus I'd like to request a `rstudio-desktop` build for win-64.

I'd be happy to help with this but I'm currently still a beginner having little experience with creating conda recipes so far so I'm still learning about it. 

Maybe @abalter, you could contribute, too, given that you seem to have given it some thorough thoughts about how to best manage R and Rstudio with conda (as I could read at [https://community.rstudio.com/t/why-not-r-via-conda/9438/4](https://community.rstudio.com/t/why-not-r-via-conda/9438/4))

Many thanks in advance for any feedback and support!

**Comment by janstrauss1** [comment]
Guess this is a good starting point: https://github.com/rstudio/rstudio/blob/17a9562b352f9a726188f570dec55e683a663c91/INSTALL

**Comment by abalter** [comment]
@janstrauss1 --- I have only ever made a conda recipe from a cran skeleton. I'm not really that great at solving complication errors.

I wonder if the starting point would be as simple as updating the rstudio version [here](https://github.com/conda-forge/rstudio-feedstock/blob/19fb1531375e896d5e9cbc855aadf848fd05468f/recipe/meta.yaml#L1) and then doing dependency whackamole. As in the versions of the prerequisites.

@gdevenyi @h-vetinari @izahn

**Comment by andyluooo** [comment]
Any updates? I really need the win64 version rstudio via conda.

**Comment by h-vetinari** [comment]
You can open a PR and help get this recipe built on windows. :)

**Comment by abalter** [comment]
@h-vetinari -- how would opening a PR help get it built? I think of a PR as saying "hey I did some work you want to include in the project" not "hey, please do some work on the project".

**Comment by h-vetinari** [comment]
> I think of a PR as saying "hey I did some work you want to include in the project" not "hey, please do some work on the project".

Exactly 😉

This is a volunteer-run do-ocracy. If you want something done, roll up your sleeves and do it. Alternatively, there are several companies offering ways to pay for getting things done in open source projects (including conda-forge).

**Comment by abalter** [comment]
@h-vetinari  that's true and also bullshit. I contribute skills where I can and have needs in other places. I don't know how to do the kind of programming for this. But I would like it. I'm sure I contribute in ways that folks who are hardcore software devs don't.

**Comment by SylvainCorlay** [comment]
> @h-vetinari -- how would opening a PR help get it built? I think of a PR as saying "hey I did some work you want to include in the project" not "hey, please do some work on the project".

Packages are built automatically when PRs to the feedstocks are merged.
